src/web/js/dashboard/config.js

```diff
@@ -1,9 +0,0 @@
-// App name in *Title Case*.
```

Comment:
This stuff was old code from when we were hosting it on my Github and planned to make it extensible to other apps. Don't need anymore if we're localizing to exclusively Pyret!